parsl/executors/high_throughput/executor.py

```diff
@@ -523,16 +524,23 @@ def hold_worker(self, worker_id: str) -> None:
         logger.debug("Sent hold request to manager: {}".format(worker_id))
 
     @property
-    def outstanding(self):
+    def outstanding(self) -> int:
+        """Returns the count of tasks outstanding across the interchange
+        and managers"""
         outstanding_c = self.command_client.run("OUTSTANDING_C")
         return outstanding_c
 
     @property
-    def connected_workers(self):
+    def connected_workers(self) -> int:
+        """Returns the count of workers across all connected managers"""
         workers = self.command_client.run("WORKERS")
         return workers
 
-    def connected_managers(self):
+    def connected_managers(self) -> List[Dict[str, typing.Any]]:
+        """Returns a list of dicts one for each connected managers.
+        The dict contains info on manager(str:manager_id), block_id,
+        worker_count, tasks(int), idle_durations(float), active(bool)
+        """
```

Comment:
This structure is frustratingly close to `ManagerRecord` in parsl/executors/high_throughput/manager_record, but not quite - ManagerRecords are transformed here: https://github.com/Parsl/parsl/blob/ce7e8f6282b1a87b56e1fd6b9d55d5a6d5f5f921/parsl/executors/high_throughput/interchange.py#L297

This docstring could be made into a very similar looking TypedDict, though, I think, which could give stronger checking of accesses to the dictionary